src/cascadia/TerminalApp/CommandPalette.cpp

```diff
@@ -474,7 +474,13 @@ namespace winrt::TerminalApp::implementation
             return;
         }
 
-        auto focusedElementOrAncestor = Input::FocusManager::GetFocusedElement(this->XamlRoot()).try_as<DependencyObject>();
+        auto root = this->XamlRoot();
+        if (root == nullptr)
```

Comment:
Why not just write `if (!root)` ("If no root")? I personally find it more readable.
BTW this kind of expression also constructs an entire `XamlRoot` object (albeit empty/nullptr) for the comparison.

Reply:
The compiler almost certainly sees right through that, right?

Reply:
Yeah you're right about that. The only advantage is in debug builds then (avoiding the construction of an `IUnknown`).